src/View/HelperRegistry.php

```diff
@@ -75,7 +75,7 @@ public function __isset(string $helper): bool
         } catch (MissingHelperException $exception) {
             $plugin = $this->_View->getPlugin();
             if (!empty($plugin)) {
-                $this->load($helper, ['className' => $plugin . '.' . $helper]);
+                $this->load($plugin . '.' . $helper);
```

Comment:
This reverts the change done in https://github.com/cakephp/cakephp/pull/17274. Are we sure we want that?

Reply:
See above. With the className not being the problem anymore, the need goes away to have this modified in such an internal way from the outside.

Reply:
@dereuromark as above, would ignoring the className differences could result in loading wrong Helper with the same base name from 2 plugins?

Reply:
Maybe, as we are only comparing them once they have been passed, which happens usually only when you have at least one plugin class in there.
So if we want this to be done cleanly, we probably should always pass in className everywhere at all times, even for non plugin ones :)

Reply:
There could also be a different related issues for className, as in some places can relate tho the same objects  both by 'Plugin.Class' format and 'Fully\Qualified\Namespaced\Class', so before comparing you might need to convert